### PR TITLE
Get user by ID function

### PR DESCRIPTION
**Pull request by ONourry**
For #85 Added getUserById function which allows us to find and return a user in the database using the userId

## successful query
http://localhost:5000/user/getUserById/b2df6835-bce6-4619-9896-643dcfb9645f

2020-12-21T07:13:35.905Z [UserSQL] info: Fetching user with id: b2df6835-bce6-4619-9896-643dcfb9645f
2020-12-21T07:13:35.916Z [userController] debug: Successfully fetched user: b2df6835-bce6-4619-9896-643dcfb9645f

returned value in postman shows the user's data

## bad query
http://localhost:5000/user/getUserById/b2df6835-bce6-4619-9896-643dcfb9645f6

2020-12-21T07:14:50.849Z [userController] info: Could not fetched user with id b2df6835-bce6-4619-9896-643dcfb9645f6: {"success":false,"data":"User does not exist"}

returned value: 
{
    "success": false,
    "error": "User does not exist"
}

### CODE REVIEW

File: back/src/route/userController.js [hunk 1]
```diff
@@ -5,12 +5,14 @@ const User = require("../objects/user");
 const fileLabel = "userController"
 const userSQL = require("../database/UserSQL");
 const utility = require("../utility");
+const UserSQL = require('../database/UserSQL');
```
Comment (kyamatek) [inline back/src/route/userController.js]: there are two userSQL

**Review by kyamatek — CHANGES_REQUESTED** [review]

**Review by ShimizuKazuki — COMMENTED** [review]
```
pbl2-back | Example app listening on port:5000
pbl2-back | (node:16) UnhandledPromiseRejectionWarning: ReferenceError: UserSQL is not defined
pbl2-back |     at getUserById (/app/src/route/userController.js:101:9)
```
Something wrong...

**Comment by ONourry** [comment]
> ```
> pbl2-back | Example app listening on port:5000
> pbl2-back | (node:16) UnhandledPromiseRejectionWarning: ReferenceError: UserSQL is not defined
> pbl2-back |     at getUserById (/app/src/route/userController.js:101:9)
> ```
> 
> Something wrong...

I had a typo. I used UserSQL instead of userSQL. The typo is now fixed

**Review by ShimizuKazuki — APPROVED** [review]
I checked

**Review by kyamatek — APPROVED** [review]

**Pull request merged**